sqlalchemy_hana/dialect.py

```diff
@@ -369,6 +369,10 @@ def visit_uuid(self, type_: types.TypeEngine[Any], **kw: Any) -> str:
         # SAP HANA has no UUID type, therefore delegate to NVARCHAR(32)
         return self._render_string_type(type_, "NVARCHAR", length_override=32)
 
+    # pylint: disable=invalid-name
```

Comment:
please remove this disable and move the function name to the `good-names` in the pyproject.toml